test/integration/scheduler_perf/config/templates/pod-with-finalizer.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: pod-
+  finalizers:
+  - test.k8s.io/finalizer
```

Comment:
Are they correctly cleaned up at the end of `SchedulingDeletedPodsWithFinalizers` even though they have a finalizer?

Reply:
Workloads are isolated. Any pods created in a previous one won't show up in the next case.

Reply:
They are also created in their own namespace for the test, right?
If so, they should have little influence when listing Pods for other tests, because they would be in a different namespace.
But ideally, we should clean them up.

Reply:
That is what I thought, but as far as I checked after @macsko's comment, looks like we re-create a cluster per test case. So, resources created in one test case cannot affect other test cases.

Reply:
Everything is cleaned up after the workload is completed. "Cluster" and etcd are newly created before each workload.

Reply:
Thanks for checking